deployment/run-with-kind.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This script utilize KIND (https://github.com/kubernetes-sigs/kind), to install Presidio
+# into docker (with k8s emulation).
+# This script is intended to get Presidio up and running in no time, so you can play with it.
+# However it is NOT intended to be installed in this manner for any workload (dev or prod).
+
+# This script should be ran ONCE. if for anyreason a retry is needed.
+# cleanup of the local environment is needed (kind delete cluster)
+
+# Clone KIND and build it (this way we eliminate the need to have GO installed)
+git clone https://github.com/kubernetes-sigs/kind.git
+cd kind
+make build
+
+# Create a KIND cluster into which we will deploy Presidio
```

Comment:
Is it possible to add notifications on the step it's currently add? some verbosity into the process as it takes a few minutes to complete

Reply:
done